### ISSUE TITLE
tableInfo.BulkConfig.UseTempDB = false;

### ISSUE THREAD

**Issue by tb0pl** [borisdj/EFCore.BulkExtensions#59]
Would it be possible for you to remove the following line of code? The code works fine without it if used within explicit transaction. The line causes issue in Sql Server since it requires account to have high privilege like db_owner or similar since normal tables are used as temp tables in merge operation and have to be created and dropped.

```c#
if (operationType != OperationType.Insert)
            {
                tableInfo.BulkConfig.UseTempDB = false; // TempDB can only be used with Insert.
                // Other Operations done with customTemp table.
                // If using tempdb[#] throws exception: 'Cannot access destination table' (gets Droped too early, probably because transaction ends)
            }
```

**Comment by borisdj** [comment]
I guess it would be.
However simple removal is not the best option.
It's better to improved the condition itself.
Either to enable explicitly forced usage of UseTempDB parameter, or I will see if it's possible to check if explicit transaction is used and set the parameter accordingly.

**Comment by borisdj** [comment]
Fix enhancement implemented and Nuget 2.1.6 published.
